src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("core.basesyntax.Bulldozer starts working");
+    }

```

Comment:
```suggestion
    public void doWork() {
        System.out.println("Bulldozer starts working");
    }

```

Reply:
It should look like this, try to resolve your naming and imports

Reply:
What do you mean by resolving imports? Creating my classes in a separate package? Thank you.